Example/Source/Controllers/CarouselController/CarouselView.swift

```diff
@@ -45,18 +45,18 @@ private typealias PrivateCarouselView = CarouselView
 private extension PrivateCarouselView {
     func activateContentViewConstraints(view: UIView) {
         var constraints = [NSLayoutConstraint]()
-        constraints.append(view.bottomAnchor.constraint(equalTo: self.bottomAnchor))
-        constraints.append(view.leadingAnchor.constraint(equalTo: self.leadingAnchor))
-        constraints.append(view.trailingAnchor.constraint(equalTo: self.trailingAnchor))
-        constraints.append(view.topAnchor.constraint(equalTo: self.topAnchor))
```

Comment:
I think explicit 'self' helps reviewer to distinct instance properties from local variables. Also it simplifies code reading in any text editor which doesn't support Swift. 
Cases like `let height = self.height(...)` will break code style consistency if we remove redundant self